modules/dynamic-provisioning-vsphere-definition.adoc

```diff
@@ -14,14 +14,10 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: <storage-class-name> <1>
-provisioner: kubernetes.io/vsphere-volume <2>
-parameters:
-  diskformat: thin <3>
+provisioner: csi.vsphere.vmware.com <2>
 ----
 <1> Name of the storage class. The persistent volume claim uses this storage class for provisioning the associated persistent volumes.
-<2> For more information about using VMware vSphere with {product-title},
+<2> For more information about using VMware vSphere csi with {product-title},
```

Comment:
If "csi" refers to "Container Storage Interface", then it should be capitalized as "CSI".